jerry-core/jerry-internal.h

```diff
@@ -36,4 +36,22 @@ jerry_dispatch_object_free_callback (ecma_external_pointer_t freecb_p,
 extern bool
 jerry_is_abort_on_fail (void);
 
+/**
+ * Snapshot header
+ */
+typedef struct
+{
+  uint32_t lit_table_size; /**< size of literal table */
+  uint32_t bytecode_size; /**< size of instructions array */
+  uint32_t idx_to_lit_map_size; /** size of idx-to-lit map */
+  uint32_t is_run_global_or_eval : 1; /**< flag, indicating whether the snapshot
+                                       *   was dumped as 'Global scope'-mode code (true)
+                                       *   or as eval-mode code (false) */
```

Comment:
I think this name is misleading, since it doesn't clear it is true on global or eval.
is_run_global would better.